packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/barriers-table.jsx

```diff
@@ -0,0 +1,76 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import { CSSTransition } from 'react-transition-group';
+import { Icon, Text } from '@deriv/components';
+import Fieldset from 'App/Components/Form/fieldset.jsx';
+
+const BarriersTable = ({
+    barrier_1,
+    setChosenBarrier,
+    is_barriers_table_expanded,
+    turbos_barrier_choices,
+    toggleBarriersTable,
+}) => {
+    const [new_barrier, setNewBarrier] = useState(barrier_1);
+    const changeBarrier = e => {
+        e.stopPropagation();
+        setNewBarrier(e.target.innerText);
+        setChosenBarrier(e.target.innerText);
+    };
+    //TODO: remove slice after adding scroll
+    const barriers_list = turbos_barrier_choices.slice(0, 16).map(barrier => (
+        <li
```

Comment:
If `li` become more complicated in future I'll provide decomposition and create a separate component.